.github/workflows/r-ci.yml

```diff
@@ -39,10 +39,18 @@ jobs:
       - name: Copy libtiledbsoma
         ## we build 'as in an R package' with all sources within apis/r/
         run: cd apis/r && ./copy_source.sh
-        
+
       - name: Test
         run: cd apis/r && tools/r-ci.sh run_tests
 
+      - name: View Install Output
+        run: cat /home/runner/work/TileDB-SOMA/TileDB-SOMA/apis/r/tiledbsoma.Rcheck/00install.out
+        if: failure() && matrix.os == 'ubuntu-latest'
+
+      - name: View Install Output
+        run: cat /Users/runner/work/TileDB-SOMA/TileDB-SOMA/apis/r/tiledbsoma.Rcheck/00install.out
+        if: failure() && matrix.os == 'macOS-latest'
+
```

Comment:
Would `cat /*/runner/work/TileDB-SOMA/TileDB-SOMA/apis/r/tiledbsoma.Rcheck/00install.out` for both allowing to collapse this into just one block?  

Or maybe also `cat ~runner/work/....`   or `cat $HOME/runner/...` ?